### PR TITLE
MetalLB make-ready: edits, structure, formatting, etc.

### PR DESCRIPTION
**Pull request by KoryKessel-Mirantis**

### CODE REVIEW

File: content/docs/configuration/metallb/_index.md [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+               frr:
+                 enabled: false
+           version: 0.14.7
+         dryRun: false
+         enabled: false
+         kind: chart
+         name: metallb
+         namespace: metallb-system
+   ```
+
```
Comment (sakshisharma84) [inline content/docs/configuration/metallb/_index.md:54]: Add a callout: FRR mode is disabled by default.

**Review by sakshisharma84 — APPROVED** [review]
lgtm!

File: content/docs/configuration/metallb/_index.md [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+ namespace: metallb-system
+spec:
+ addresses:
+ - 192.155.10.0/24
+---
+apiVersion: metallb.io/v1beta1
+kind: L2Advertisement
+metadata:
+ name: empty
+ namespace: metallb-system
```
Comment (quadespresso) [inline content/docs/configuration/metallb/_index.md:198]: Please change the indent from 1 space to 2 spaces, ie,

```yaml
apiVersion: metallb.io/v1beta1
kind: IPAddressPool
metadata:
  name: example1
  namespace: metallb-system
spec:
  addresses:
  - 192.168.10.0/24
  - 192.168.1.0/24
---
apiVersion: metallb.io/v1beta1
kind: IPAddressPool
metadata:
  name: example2
  namespace: metallb-system
spec:
  addresses:
  - 192.155.10.0/24
---
apiVersion: metallb.io/v1beta1
kind: L2Advertisement
metadata:
  name: empty
  namespace: metallb-system
```

**Review by quadespresso — CHANGES_REQUESTED** [review]

File: content/docs/configuration/metallb/_index.md [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+| name                                | Name of the add-on.                                     | metallb                                                                                   |
+| namespace                           | Namespace used for<br>deploying MetalLB.                   | metallb-system                                                                            |
+| chart.name                          | Name of the MetalLB Helm chart.                         | metallb                                                                                   |
+| chart.repo                          | MetalLB Helm repository.                                | https://metallb.github.io/metallb                                                         |
+| chart.version                       | Version of the MetalLB Helm chart.                      | 0.14.7                                                                                    |
+| chart.values.<br>controller.tolerations | Tolerations for the<br>MetalLB controller pod.             | YAML: <br><br> - key: node-role.kubernetes.io/master<br>   &nbsp; operator: Exists <br>  &nbsp;   effect: NoSchedule |
```
Comment (quadespresso) [inline content/docs/configuration/metallb/_index.md:99]: Unify into a single string, and add backticks: `chart.values.controller.tolerations`

File: content/docs/configuration/metallb/_index.md [hunk 4]
```diff
@@ -0,0 +1,215 @@
...
+| chart.values.<br>speaker.frr.enabled    | Enablement state of FRR<br>with regard to MetalLB speaker. | false                                                                                     |
```
Comment (quadespresso) [inline content/docs/configuration/metallb/_index.md:100]: Unify into a single string and add backticks: `chart.values.speaker.frr.enabled`

and add backticks: `false`

**Review by quadespresso — CHANGES_REQUESTED** [review]

File: content/docs/configuration/metallb/_index.md (hunk 2 above)
Comment (KoryKessel-Mirantis) [reply content/docs/configuration/metallb/_index.md:198]: Done.

**Review by quadespresso — APPROVED** [review]

File: content/docs/configuration/metallb/_index.md [hunk 5]
```diff
@@ -0,0 +1,215 @@
...
+
+MetalLB is a load balancer designed for bare metal Kubernetes clusters that uses standard routing protocols.
+
+## Prerequisites
+
+- An MKE cluster that does not already have load-balancing functionality.
+
+- A MetalLB-compatible cluster network configuration.
+
+- `kube-proxy` running in `iptables` mode.
```
Comment (MagdaDziadosz) [inline content/docs/configuration/metallb/_index.md:14]:
```suggestion
- A `kube-proxy` component running in `iptables` mode.
```

Listed as component in the official doc: https://kubernetes.io/docs/concepts/overview/components/#node-components

File: content/docs/configuration/metallb/_index.md [hunk 6]
```diff
@@ -0,0 +1,215 @@
...
+
+- The absence of any cloud provider configuration.
+
+## Configuration
+
+You configure MetalLB through the `Addons` section of the MKE 4 configuration
```
Comment (MagdaDziadosz) [inline content/docs/configuration/metallb/_index.md:20]:
```suggestion
You can configure MetalLB through the `Addons` section of the MKE 4 configuration
```

File: content/docs/configuration/metallb/_index.md [hunk 7]
```diff
@@ -0,0 +1,215 @@
...
+file. The function is disabled by default, and thus to use MetalLB you must set
+the `enabled` parameter to `true`.
+
+1. Obtain the default configuration file.
```
Comment (MagdaDziadosz) [inline content/docs/configuration/metallb/_index.md:24]:
```suggestion
1. Obtain the default configuration file:
```

File: content/docs/configuration/metallb/_index.md [hunk 8]
```diff
@@ -0,0 +1,215 @@
...
+An MKE version comparison for MetalLB configuration parameters is offered in the following table:
+
+| MKE-3                                                          | MKE-4                                   |
+|----------------------------------------------------------------|-----------------------------------------|
+| `[cluster_config.metallb_config.enabled]`                        | `addons.metallb.enabled`                  |
+| `[[cluster_config.metallb_config.metallb_ip_addr_pool]]`         | `Deprecated`                            |
+
+## IP address pool configuration
+
+  <callout type="info"> Support for IP address pool configuration is deprecated in MKE 4, and thus you must now independently manage the configuration. </callout>
```
Comment (MagdaDziadosz) [inline content/docs/configuration/metallb/_index.md:111]:
```suggestion
  <callout type="info"> Support for IP address pool configuration is deprecated in MKE 4, and thus you must now manage the configuration independently. </callout>
```

Check if the note is wrapped in the output, if not, divide it between two lines.

File: content/docs/configuration/metallb/_index.md [hunk 9]
```diff
@@ -0,0 +1,215 @@
...
+name: example1
+ipaddress: [192.168.10.0/24 192.168.1.0/24]
+
+name: example2
+ipaddress: [192.155.10.0/24]
+
+Please make sure that you create these pools after MKE4 installation is complete.
+```
+
+Refer to the upgrade summary for guidance in the creation of IP address pools. To create the pools, you can use the template that follows or consult the official MetalLB documentation [Layer 2 configuration](https://metallb.universe.tf/configuration/#layer-2-configuration) for assistance.
```
Comment (MagdaDziadosz) [inline content/docs/configuration/metallb/_index.md:172]:
```suggestion
Refer to the upgrade summary for guidance in the creation of IP address pools. To create the pools,
you can use the template that follows or consult the official MetalLB documentation
[Layer 2 configuration](https://metallb.universe.tf/configuration/#layer-2-configuration) for assistance.
```

File: content/docs/configuration/metallb/_index.md [hunk 10]
```diff
@@ -0,0 +1,215 @@
...
+file. The function is disabled by default, and thus to use MetalLB you must set
+the `enabled` parameter to `true`.
+
+1. Obtain the default configuration file.
+
+   ```bash
+   mkectl init
+   ```
+
+   The example that follows presents the MetalLB configuration section as it is used to deploy MetalLB as a Helm chart within the cluster:
```
Comment (MagdaDziadosz) [inline content/docs/configuration/metallb/_index.md:30]:
```suggestion
   The following example presents the MetalLB configuration section used to deploy MetalLB as a Helm chart within the cluster:
```

File: content/docs/configuration/metallb/_index.md [hunk 11]
```diff
@@ -0,0 +1,215 @@
...
+| `[cluster_config.metallb_config.enabled]`                        | `addons.metallb.enabled`                  |
+| `[[cluster_config.metallb_config.metallb_ip_addr_pool]]`         | `Deprecated`                            |
+
+## IP address pool configuration
+
+  <callout type="info"> Support for IP address pool configuration is deprecated in MKE 4, and thus you must now independently manage the configuration. </callout>
+
+### Create IP address pools in a fresh installation
+
+  For information on how to create IP address pools, refer to the official MetalLB documentation, [Layer 2 configuration](https://metallb.universe.tf/configuration/#layer-2-configuration).
```
Comment (MagdaDziadosz) [inline content/docs/configuration/metallb/_index.md:115]:
```suggestion
  For information on how to create IP address pools, refer to the official MetalLB documentation,
  [Layer 2 configuration](https://metallb.universe.tf/configuration/#layer-2-configuration).
```

**Review by MagdaDziadosz — CHANGES_REQUESTED** [review]

File: content/docs/configuration/metallb/_index.md (hunk 5 above)
Comment (KoryKessel-Mirantis) [reply content/docs/configuration/metallb/_index.md:14]: Yes, kube-proxy is a Kubernetes component, but drawing it out as such reads _awkward_. I would go with 'instance' before 'component'. That said, though, it does bring up a question. @sakshisharma84, should be saying "A kube-proxy instance running in `iptables` mode on each cluster node."?

File: content/docs/configuration/metallb/_index.md (hunk 10 above)
Comment (KoryKessel-Mirantis) [reply content/docs/configuration/metallb/_index.md:30]: The edit changes the emphasis of the sentence.

File: content/docs/configuration/metallb/_index.md [hunk 12]
```diff
@@ -0,0 +1,219 @@
...
+
+- The absence of any cloud provider configuration.
+
+## Configuration
+
+MetalLB is configured through the `Addons` section of the MKE 4 configuration
+file. The function is disabled by default, and thus to use MetalLB you must set
+the `enabled` parameter to `true`.
+
+1. Obtain the default configuration file:
```
Comment (KoryKessel-Mirantis) [inline content/docs/configuration/metallb/_index.md:24]:
```suggestion
1. Obtain the default MKE configuration file:
```

File: content/docs/configuration/metallb/_index.md [hunk 13]
```diff
@@ -0,0 +1,219 @@
...
+
+- A MetalLB-compatible cluster network configuration.
+
+- `kube-proxy` running in `iptables` mode.
+
+- The absence of any cloud provider configuration.
+
+## Configuration
+
+MetalLB is configured through the `Addons` section of the MKE 4 configuration
```
Comment (KoryKessel-Mirantis) [inline content/docs/configuration/metallb/_index.md:20]:
```suggestion
MetalLB is configured through the `addons` section of the MKE 4 configuration
```

**Review by MagdaDziadosz — APPROVED** [review]

**Pull request merged**